completions/sv.zsh

```diff
@@ -35,7 +35,11 @@ cmds)
         check
     ret=0;;
 args)
-    services=( /var/service/*(-/N:t) )
+    if [[ $BUFFER == sudo* ]] then
```

Comment:
I think it makes sense to also detect at least `doas`, right?

Reply:
I didn't know about `doas`, good point. I added it and also added `su -c`.